app/src/ui/branches/branches-container.tsx

```diff
@@ -91,6 +97,13 @@ export class BranchesContainer extends React.Component<
     )
   }
 
+  private renderBranch = (
```

Comment:
Because we're not using any props or local variables for the renderBranch method, we can probably import `renderDefaultBranch` and use it inside the `BranchList` component. Feels like there's nothing unique we're doing at this higher level.